.gitignore

```diff
@@ -8,4 +8,7 @@ lib
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
-!.yarn/versions
+
+# We are using the `version` plugin just to do a bump commit.
+# We don't need the entire https://yarnpkg.com/features/release-workflow feature set.
```

Comment:
Not sure to understand this comment. What's inside `.yarn/versions` and why does it need to be versioned now?